Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1165,6 +1165,11 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+When the fixed cookie value is not used, a service affinity of a service with multiple
+ports is per service IP and port. Meaning that if a request from the same source
+to the same service but different ports is sent, each request might stick to
+different service endpoints.
```

Comment:
I'm having troubles parsing this sentence. "each request might stick to different service endpoints", different from what?

Reply:
What's about ", each request might stick to different service endpoints" => ", for each service connection a different service endpoint might be selected"?

Reply:
Meaning that if a request from the same source to the same service but with a different destination ports is sent, each connection might be selected to a different service endpoint.?

But also it gives the impression that sessionAffinity will only work for 1 port out of the N ports a service has, don't you think?

Reply:
Pasting (https://github.com/cilium/cilium/pull/16954#discussion_r673905230) it here for the discussion:

> Meaning that if a request from the same source to the same service but with a different destination ports is sent, each connection might be selected to a different service endpoint.?
>
> But also it gives the impression that sessionAffinity will only work for 1 port out of the N ports a service has, don't you think?

To me it's not, otherwise I would not write it :-) @qmonnet How does it read for you?

Reply:
`a service affinity of a service with multiple ports is per service IP and port`: This disambiguated the example in my mind, but if you focus just on the second sentence (`Meaning that ...`), I can see how it might read that successive connections with the same couple IP/port might end up on distinct endpoints. I don't really understand André's “it gives the impression that sessionAffinity will only work for 1 port out of the N ports a service has”, though.

How about something like this:
```
When the fixed cookie value is not used, a service affinity of a service with
multiple ports is per service IP and port. Meaning that all requests for a
given service sent from the same source *and to the same port* will be routed
to the same service endpoints; but two requests for the same service, sent from
the same source *but to different ports* may be routed to distinct service
endpoints.
```

Reply:
SGTM after applying this - *and to the same port* => *and to the same service port* and *but to different ports* => *but to different service ports*. @aanm Thoughts?

Reply:
LGTM